src/galileo/resources/models/message_role.py

```diff
@@ -1,13 +1,3 @@
-from enum import Enum
-
-
-class MessageRole(str, Enum):
-    AGENT = "agent"
-    ASSISTANT = "assistant"
-    FUNCTION = "function"
-    SYSTEM = "system"
-    TOOL = "tool"
-    USER = "user"
-
-    def __str__(self) -> str:
-        return str(self.value)
+# flake: noqa: F401
+# ruff: noqa: F401
+from ..models.all_galileo_core_models import MessageRole
```

Comment:
Link individual model file to import the right definition from all model file to ensure generated clients can still use these models.